web/public/routes/home/home.css

```diff
@@ -52,6 +55,8 @@
 
 .image-box {
   margin-bottom: 20px;
+  padding-bottom: 20px;
+  border-bottom: 1px solid #eaeaea;
```

Comment:
@rvennam 

the color definition should be moved to darkvision-light.css and its counterpart created in darkvision-dark.css. Check if you could add the "border-box" to the element. It is the light border color used in other places in the ui.

no color in darkvision.css, home.css or video.css to support the theme switching.